### ISSUE TITLE
Join section suggestions

### ISSUE THREAD

**Issue by cjyabraham** [cncf/cncf.io#130]
At the bottom of the [join page](https://dev-cncfci.pantheonsite.io/about/join/), what about doing the following:
- make all three buttons the same width
- number the three sections so that they seem like either/or alternatives

<img width="856" alt="Screen Shot 2020-06-05 at 3 22 38 PM" src="https://user-images.githubusercontent.com/463160/83848461-81769800-a740-11ea-89bc-59d16655ca93.png">

**Comment by thetwopct** [comment]
I've rejigged them so they stagger in a nice manner

![20200605132603-Join CNCF | Cloud Native Computing Foundation](https://user-images.githubusercontent.com/10615884/83876095-2dfc4e00-a730-11ea-905e-e32e2f412f2d.jpg)

When playing with this, I found this way the most button like. If they are all the same width they don't tend to look like buttons, and stagger strangely as the content inside is centered.

It's also a shame a cncf.io redirect is used for the href - even thought they are external - otherwise each would have an external icon automatically inserted in the button.

![20200605132826-Test | Cloud Native Computing Foundation](https://user-images.githubusercontent.com/10615884/83876235-7156bc80-a730-11ea-994a-c03e369b7a96.jpg)
